.github/workflows/test.yml

```diff
@@ -44,26 +44,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
-          - java: '11'
-            mongo: 'mongodb-linux-x86_64-3.6.23'
-            minio: '2019-05-23T00-29-34Z'
-            wired_tiger: 'true'
-            ant_test: 'test_quick_coverage'
-          # the current production setup
-          - java: '11'
-            mongo: 'mongodb-linux-x86_64-3.6.13'
-            minio: '2019-05-23T00-29-34Z'
-            wired_tiger: 'false'
-            ant_test: 'test_quick_coverage'
+        mongo: ['3.6.13', '3.6.23']
+        test_cmd: ['test_quick_coverage', 'NoLongTests']
```

Comment:
`test_quick_coverage` is the old ant tests, `NoLongTests` is the new gradle tests